amplify/backend/function/fetchDomainBalance/src/event.json

```diff
@@ -0,0 +1,5 @@
+{
+  "key1": "value1",
+  "key2": "value2",
+  "key3": "value3"
+}
```

Comment:
I'm a bit lost on what this file does?

Reply:
This file is generated when we create the lambda function in case we want to locally trigger the lambda function
for eg. if you're using the sam cli, you can run `sam local invoke fetchDomainBalance-{ENV_HERE} -e event.json`
But as I haven't seen any of the `event.json` files to be kept in sync with the actual parameters need by the lambda function, I haven't updated this file 🤔

Reply:
Ah okay, understood! You can ignore me then :)

Reply:
We usually just get rid of this auto generated file